app/assets/javascripts/discourse/app/initializers/animated-images.js

```diff
@@ -61,6 +96,33 @@ export default {
       });
 
       api.cleanupStream(_cleanUp);
+
+      // paused on load when prefers-reduced-motion is active, no need for blur/focus events
+      if (!prefersReducedMotion()) {
+        const images = "img.animated:not(.manually-paused)";
+
+        window.addEventListener("blur", () => {
```

Comment:
we could probably remove these listeners in the `teardown()` hook of initializer for extra safety.